examples/Benchmark/Performance/MatrixAssembly/MatrixAssembly_direct_blocs.scn

```diff
@@ -27,7 +27,7 @@ The differences are in the way the global system matrix is built and solved:
             and provided to the linear solver.
             The template CompressedRowSparseMatrixMat3x3d indicates that the matrix is defined by 3x3 blocs.
          -->
-        <SparseLDLSolver template="CompressedRowSparseMatrixMat3x3d"/>
+        <SparseLUSolver template="CompressedRowSparseMatrixMat3x3d"/>
```

Comment:
I am not sure this is change you wanted to push

Reply:
It was for a test, i forgot to leave it out